content/en/docs/distributions/gke/pipelines/preemptible.md

```diff
@@ -49,41 +46,61 @@ The following sections contain more detail about the above steps.
 
 ### 1. Create a node pool with preemptible VMs
 
-Use the `gcloud` command to
-[create a node pool](https://cloud.google.com/sdk/gcloud/reference/container/node-pools/create).
-The following example includes placeholders to illustrate the important
-configurations:
+Create a `preemptible-nodepool.yaml` as below and fulfill all placerholder content:
+
+```
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerNodePool
+metadata:
+  labels:
+    kf-name: KF_NAME # {"$kpt-set":"name"}
+  name: PREEMPTIBLE_CPU_POOL
+  namespace: KF_PROJECT # {"$kpt-set":"project"}
+spec:
+  location: LOCATION # {"$kpt-set":"location"}
+  initialNodeCount: 1
+  autoscaling:
+    minNodeCount: 0
+    maxNodeCount: 5
+  nodeConfig:
+    machineType: n1-standard-4
+    diskSizeGb: 100
+    diskType: pd-standard
+    preemptible: true
+    taint:
+    - effect: NO_SCHEDULE
+      key: preemptible
+      value: "true"
+    oauthScopes:
+    - "https://www.googleapis.com/auth/logging.write"
+    - "https://www.googleapis.com/auth/monitoring"
+    - "https://www.googleapis.com/auth/devstorage.read_only"
+    serviceAccountRef:
+      external: KF-NAME-vm@KF-PROJECT.iam.gserviceaccount.com # {"$kpt-set":"vm-sa-ref"}
+    metadata:
+      disable-legacy-endpoints: "true"
+  management:
+    autoRepair: true
+    autoUpgrade: true
+  clusterRef:
+    name: KF_NAME # {"$kpt-set":"name"}
+    namespace: KF_PROJECT # {"$kpt-set":"project"}
+```
 
-    gcloud container node-pools create PREEMPTIBLE_CPU_POOL \
-    	--cluster=CLUSTER_NAME \
-        --enable-autoscaling --max-nodes=MAX_NODES --min-nodes=MIN_NODES \
-        --preemptible \
-        --node-taints=preemptible=true:NoSchedule \
-        --service-account=DEPLOYMENT_NAME-vm@PROJECT_NAME.iam.gserviceaccount.com
```

Comment:
I just realized, if people are still using AI Platform Pipelines or Kubeflow Pipelines standalone. They would still need to use these commands to create node pools. It's up to you for consideration, maybe we need both.

Reply:
I have included both approaches for the completeness of guidance.